CSharpOOP4/Ball.cs

```diff
@@ -0,0 +1,24 @@
+﻿namespace Kristina_Kulich__Application.CSharpOOP4;
+
+public class Ball : Shape
+{
+    public string _name;
+    public double _radius;
+
+
+    public Ball(string name, double radius)
```

Comment:
add :base(name) to constructor. It allows you to use name parameter in base class.

Reply:
fixed